models/migrations/migrations.go

```diff
@@ -18,7 +18,7 @@ import (
 	"github.com/Unknwon/com"
 	"github.com/go-xorm/xorm"
 	gouuid "github.com/satori/go.uuid"
-	"gopkg.in/ini.v1"
+	ini "gopkg.in/ini.v1"
```

Comment:
As long as it's done in a separate commit it's OK. It's an goimports correction.